src/service/SentenceFinder.java

```diff
@@ -0,0 +1,19 @@
+package service;
```

Comment:
Need one more solution with arrays and without split

Reply:
Working on that